HealthSDK/GiniHealthSDK/Sources/GiniHealthSDK/Core/GiniHealth.swift

```diff
@@ -70,7 +72,7 @@ public struct DataForReview {
      
      - parameter giniApiLib: GiniHealthAPI initialized with client's credentials
      */
-    public init(with giniApiLib: GiniHealthAPI){
+    public init(with giniApiLib: HealthAPI){
```

Comment:
[Clarification]: Could you please double check I think we should keep here `GiniHealthAPI` ?